packages/web-app/.env.sample

```
@@ -1,4 +1,5 @@
-NEXT_PUBLIC_CHAIN="goerli"
-IPFS_API_TOKEN=""
-GRAPH_API_URL=""
-NEXT_PUBLIC_ALCHEMY_API_KEY=""
\ No newline at end of file
+NEXT_PUBLIC_CHAIN="mainnet"
+NEXT_PUBLIC_NFTSTORAGE_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJkaWQ6ZXRocjoweDI2NmFCZGE3QWNmODc0NTlBRTJEOTg5RjM4OWU1N0I0MmJCNkNkNzYiLCJpc3MiOiJuZnQtc3RvcmFnZSIsImlhdCI6MTcwOTIwOTQwNDM3MiwibmFtZSI6IkRhbyJ9.YsuBC1iXGJ7Ip_VkFjsZSzHQSOkKRMyaCw4eq_j5f-4
+NEXT_PUBLIC_GRAPH_API_URL=https://api.thegraph.com/subgraphs/name/nation3/nation3-agreements-mainnet
+NEXT_PUBLIC_GRAPH_API_URL_SEPOLIA=https://api.thegraph.com/subgraphs/name/nation3/nation3-agreements-sepolia
```

I suggest we rename this variable to `NEXT_PUBLIC_GRAPH_API_URL` and put it in a file called `.env.sepolia`
```suggestion
NEXT_PUBLIC_GRAPH_API_URL=https://api.thegraph.com/subgraphs/name/nation3/nation3-agreements-sepolia
```